Updated stations, stop and facilities using the PHP scripts.

Bertware opened the pull request:
Automated update using GTFS data, the NMBS website, and our GTFS parser/website scraper scripts.

- 5 new stations (see below)
- updated avg_stop_times
- updated facilities (addresses, opening times)

New stations:
- Aulnoye aymeries
- Quevy-frontiere
- Melle PW (Polyvalente werkplaats, open to public on March 30th 2019: https://www.belgiantrain.be/nl/about-sncb/themes/opendoors)
- Petange
- Maubeuge

Bertware commented:
CI seems broken, I think I've seen this error before but can't rember how we fixed it, @pietercolpaert any idea?

DylanVanAssche commented:
> CI seems broken, I think I've seen this error before but can't rember how we fixed it, @pietercolpaert any idea?

Seems that the CI wants to clone the repo using SSH. If the CI doesn't have access to the repo through its own SSH key, the CI can't clone the repo. For submodules for example, I always use HTTPS as clone URL to avoid this issue in Travis CI.

Bertware commented:
A while ago I added the repo location to the composer file in order to allow for quicker updates (otherwise it can take a while to propagate to the composer repositories before it becomes available to, for example, the iRail API), looks like that was somehow the cause, fixed it by removing this configuration part again. @DylanVanAssche care to review the data changes? Mostly station names etc, in case I missed something there.

Bertware commented:
Included a fix for #145

DylanVanAssche commented:
@Bertware Sure! I will review the data ASAP :)

DylanVanAssche commented on facilities.csv:
```diff
@@ -1,4 +1,5 @@
 URI,name,street,zip,city,ticket_vending_machine,luggage_lockers,free_parking,taxi,bicycle_spots,blue-bike,bus,tram,metro,wheelchair_available,ramp,disabled_parking_spots,elevated_platform,escalator_up,escalator_down,elevator_platform,audio_induction_loop,sales_open_monday,sales_close_monday,sales_open_tuesday,sales_close_tuesday,sales_open_wednesday,sales_close_wednesday,sales_open_thursday,sales ...
+http://irail.be/stations/NMBS/008729560,Aulnoye aymeries,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Capital letter? Dash?
https://en.wikipedia.org/wiki/Aulnoye-Aymeries

Bertware replied:
Will fix with a capital letter and a dash! https://media0.giphy.com/media/3o7aCRloybJlXpNjSU/giphy.gif?cid=3640f6095c98fc87416b5a2f4db13f66

DylanVanAssche commented on facilities.csv:
```diff
@@ -559,10 +565,10 @@ http://irail.be/stations/NMBS/008833159,Sint-Joris-Weert,Stationsstraat Zn.,3051
 http://irail.be/stations/NMBS/008821543,Sint-Mariaburg,Lodewijk Andessonstraat,2180,Antwerpen (ekeren),1,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008812245,Sint-Martens-Bodegem,Stationsplein,1700,Sint Martens Bodegem - Dilbeek,1,0,1,0,0,0,0,0,0,0,0,0,1,0,0,0,0,,,,,,,,,,,,,,
-http://irail.be/stations/NMBS/008894508,Sint-Niklaas,Leopold Ii Laan 2,9100,Sint-niklaas,1,1,1,1,1,1,1,0,0,0,1,9,0,1,0,1,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,06:00,20:00,06:00,20:00
+http://irail.be/stations/NMBS/008894508,Sint-Niklaas,Leopold Ii Laan 2,9100,Sint-niklaas,1,0,1,1,1,1,1,0,0,0,1,9,0,1,0,1,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,06:00,20:00,06:00,20:00
```
Sint-Niklaas instead of Sint-niklaas

mstuyts replied:
It probably should be Leopold II Laan

Bertware replied:
@DylanVanAssche think the current casing is correct, according to wikipedia and the communal website https://nl.wikipedia.org/wiki/Sint-Niklaas
https://www.sint-niklaas.be/

I'll fix the Leopold II laan, this file is created programmatically so the PHP script will need fixing as well

DylanVanAssche replied:
I mean this one: `Leopold Ii Laan 2,9100,Sint-niklaas`. @Bertware

Bertware replied:
Got it!

pietercolpaert commented on facilities.csv:
```diff
@@ -657,7 +663,7 @@ http://irail.be/stations/NMBS/008833274,Zichem,Ernest Claesstraat Zn,3271,Zichem
+http://irail.be/stations/NMBS/008895208,Zottegem,Stationsplein 12,9620,Zottegem,1,0,1,0,1,1,1,0,0,1,1,2,0,0,0,0,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,07:15,14:30,07:15,14:30
 http://irail.be/stations/NMBS/008891611,Zwankendamme,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008894821,Zwijndrecht,Fortlaan 1,2070,Zwijndrecht,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008883212,Écaussinnes,Avenue De La DÉportation,7190,Ecaussinnes,1,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,
```
Déportation?

Bertware replied:
Nice catch, will fix, and as this file is created programmatically the PHP script will need fixing as well
